ext/search_kit/ang/crmSearchAdmin/displays/searchAdminDisplayEntity.html

```diff
@@ -37,6 +37,11 @@
     <option value="OR">{{:: ts('Or') }}</option>
   </select>
 </div>
+<div class="form-inline">
```

Comment:
@totten if you scroll up a little in this file, there is some UI specific to Tables that probably doesn't apply to Views. E.g. the "Auto-Refresh"  option actually creates a scheduled job.

Reply:
Thanks. I moved the TABLE/VIEW selector and made the refresh-related fields conditional.